### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by danielschonfeld**
This should fill the missing gap cited by #8250

### CODE REVIEW

**Comment by danielschonfeld** [comment]
I signed it!

File: cluster/addons/dns/README.md [hunk 1]
```diff
@@ -58,6 +58,37 @@ Salt.  You will need to replace the `{{ <param> }}` blocks with your own values
...
+    cluster: local
+  name: service-account-context
+current-context: service-account-context
+```
+Next, you will want to convert this `Config` into a base64 string.  You can do that easily in linux/mac by invoking `base64 -i config.yaml`.  That should give you a base64 string, which you will then be able to copy into the `Secret` resource file.
+
+Finally, we create the `Secret` resource file and make the key name 'kubeconfig' as that is the filename that will be used by kube2sky.
+
+```
+apiVersion: v1beta1
```
Comment (bgrant0607) [inline cluster/addons/dns/README.md]: Please update to v1beta3. We are removing v1beta1 imminently.

For a v1beta3 secret example, see https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/secrets.md#use-case-pod-with-ssh-keys
(the first example in that doc is wrong)

**Comment by danielschonfeld** [comment]
@cjcullen yes you're absolutely right.  in fact to make this work on my own network, utilizing the insecure api server port, I had to override the master url flag for kube2sky.  I'll have to update this PR.

@bgrant0607 noted, will do.  I'll also have to update the example for either forcing the master URL flag as was referenced above or to utilize a token.

**Comment by cjcullen** [comment]
I believe this PR has been usurped (and made out-of-date) by the DNS changes by @thockin in #10150.

Thanks for the detailed doc, and sorry things churned out from under you!

**Pull request closed without merging**